src/structures/VoiceChannel.js

```diff
@@ -71,6 +71,15 @@ class VoiceChannel extends GuildChannel {
     return super.deletable && this.permissionsFor(this.client.user).has(Permissions.FLAGS.CONNECT, false);
   }
 
+  /**
+   * Whether the channel is editable by the client user
+   * @type {boolean}
+   * @readonly
+   */
+  get editable() {
+    return super.manageable && this.permissionsFor(this.client.user).has(Permissions.FLAGS.CONNECT, false);
```

Comment:
Maybe `this` instead of `super` since it's not overriding the method

Reply:
```suggestion
    return this.manageable && this.permissionsFor(this.client.user).has(Permissions.FLAGS.CONNECT, false);
```